src/unstract/sdk/vector_db.py

```diff
@@ -21,12 +21,9 @@
 class ToolVectorDB:
     """Class to handle VectorDB for Unstract Tools."""
 
-    def __init__(self, tool: BaseTool, tool_settings: dict[str, str] = {}):
+    def __init__(self, tool: BaseTool):
         self.tool = tool
         self.vector_db_adapters = adapters
-        self.vector_db_adapter_instance_id = tool_settings.get(
-            ToolSettingsKey.VECTOR_DB_ADAPTER_ID
-        )
```

Comment:
@chandrasekharan-zipstack since we are removing an instance variable hope you have double checked that this is not used outside this